KubeArmor/main_test.go

```diff
@@ -28,6 +30,6 @@ func TestMain(t *testing.T) {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 
 	// Set os args to set flags in main
-	os.Args = []string{"cmd", "-cluster", *clusterPtr, "-gRPC", *gRPCPtr, "-logPath", *logPathPtr, "-enableHostPolicy", *enableHostPolicyPtr, "-enableEnforcerPerPod", *enableEnforcerPerPodPtr}
+	os.Args = []string{"cmd", "-cluster", *clusterPtr, "-gRPC", *gRPCPtr, "-logPath", *logPathPtr, "-enableHostPolicy", strconv.FormatBool(*enableHostPolicyPtr), "-enableEnforcerPerPod", strconv.FormatBool(*enableEnforcerPerPodPtr)}
```

Comment:
is "strconv.FormatBool" required? those are already the bool types, aren't they?

Reply:
We are converting bool to string. 
We need it to be string since [`os.Args`](https://pkg.go.dev/os#Args) is a string array.

Reply:
But we need the flag to be boolean, since if we set it to string it's behavior changes and it demands an argument.

Reply:
then, it should be something like the below since the arguments will be parsed by the main function, shouldn't it?
os.Args = []string{"cmd", "-cluster", *clusterPtr, "-gRPC", *gRPCPtr, "-logPath", *logPathPtr, "-enableHostPolicy", "-enableEnforcerPerPod"}

Reply:
`-enableHostPolicy` == `-enableHostPolicy true`

We will have to add if checks to set it to false, or not add it to the string array.